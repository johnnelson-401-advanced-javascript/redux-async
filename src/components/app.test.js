import React from 'react';
import { shallow } from 'enzyme';
import App from './app';


describe('App component', () => {
  it('Renders and App component', () => {
    const characters = [{
      _id: '123',
      name: 'name',
      photoUrl: 'nope'
    }];
    const wrapper = shallow(<App characters={characters} />);
    expect(wrapper).toMatchSnapshot();
  });
  
});
