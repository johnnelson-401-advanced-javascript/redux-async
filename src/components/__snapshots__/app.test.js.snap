// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Renders and App component 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      path="/"
    />
  </Switch>
</BrowserRouter>
`;
